grid-compose/internal/app.go

```diff
@@ -0,0 +1,372 @@
+package internal
```

Comment:
a better structure for this file could be a separate package `internal/app` have the `app.go` file for the app struct & factory method. and a file for each command with its utils `ps.go`, `down.go`, ...